templates/core/terraform/api-webapp.tf

```diff
@@ -44,6 +44,7 @@ resource "azurerm_linux_web_app" "api" {
     "MANAGED_IDENTITY_CLIENT_ID"                     = azurerm_user_assigned_identity.id.client_id
     "TRE_ID"                                         = var.tre_id
     "RESOURCE_LOCATION"                              = azurerm_resource_group.core.location
+    "ENABLE_SWAGGER"                                 = var.enable_swagger
```

Comment:
Github actions are using secrets and it's not ideal for us to put values of true/false there as it obfuscate that value across all log outputs. 
Can you do a change that would effectively allow any value other than false to turn this option on?

Reply:
let's talk have a quick chat about this, what do you mean by obfuscating the logs?
why boolean values in particular cause more obfuscation?

Reply:
update: we've decided to keep it as boolean for now